src/gui/optionsdialog.ui

```diff
@@ -2485,17 +2485,27 @@
                  </property>
                  <item>
                   <property name="text">
-                   <string>Pause them</string>
+                   <string>Pause torrent</string>
```

Comment:
I think these should all be plural (i.e. `Pause torrents`) given the sentence `Seed torrents until their seeding time...`.

Reply:
The only reason i changed them is because "Delete torrents + folder" would look weird IMO.